autonomy/cli/hash.py

```diff
@@ -129,8 +126,6 @@ def update_hashes(  # pylint: disable=too-many-locals
                 if vendor is not None and package_id.author != vendor:
                     continue  # pragma: nocover
                 package_hashes[key] = package_hash
-
-        to_csv(package_hashes, packages_dir / HASHES_FILE)
```

Comment:
So the update won't take effect? where is it being saved?

Reply:
@angrybayblade

Reply:
Won't be saved, will be done on `autonomy packages lock`